src/core/client.ts

```diff
@@ -269,7 +269,12 @@ class ApiClient<R extends RawApi> {
             ? createFormDataPayload(payload, (err) => streamErr.catch(err))
             : createJsonPayload(payload);
         const sig = controller.signal;
-        const options = { ...opts.baseFetchConfig, signal: sig, ...config };
+        const options = {
+            ...opts.baseFetchConfig,
+            // deno-lint-ignore no-explicit-any
```

Comment:
Do not use `as any` and ignore the lint to work around a type issue. If it’s a type issue tell that directly to typescript via ts-ignore-error

Reply:
This is not related to this PQ. I can fix it separately at a later point.

Reply:
This PQ adds the as any so I think it’s done in this one?

Reply:
It no longer adds this. This is outdated by now.